setup.py

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env python
```

Comment:
Switching to a pure `pyproject.toml` configuration instead of using `setup.cfg` and `setup.py` should be considered.

Reply:
https://github.com/spacetelescope/romanisim/pull/13